routers/api/v1/repo/pull.go

```diff
@@ -102,7 +103,7 @@ func ListPullRequests(ctx *context.APIContext, form api.ListPullRequestsOptions)
 			ctx.Error(http.StatusInternalServerError, "GetHeadRepo", err)
 			return
 		}
-		apiPrs[i] = prs[i].APIFormat()
+		apiPrs[i] = convert.ToPullRequest(prs[i])
```

Comment:
I'm OK with this, but we should consider naming these functions `ToPullRequestAPI` or `ToAPIPullRequest` in case there will be conversions in the other direction or for other purposes.